superdesk/media/crop.py

```diff
@@ -182,13 +182,44 @@ def create_multiple_crops(self, updates, original):
         update_renditions = updates.get('renditions', {})
         if original.get(ITEM_TYPE) == CONTENT_TYPE.PICTURE and update_renditions:
             renditions = original.get('renditions', {})
+            poi = updates.get('poi', {})
             original_copy = deepcopy(original)
+            if poi:
+                for crop_name in renditions:
+                    self._set_crop_poi(renditions, crop_name, poi)
             for key in update_renditions:
                 if self.get_crop_by_name(key):
-                    renditions, crop_created = self.create_crop(original_copy, key, update_renditions.get(key, {}))
+                    renditions, crop_created = self.create_crop(original_copy, key,
+                                                                update_renditions.get(key, {}), poi)
 
             updates['renditions'] = renditions
 
+    def _set_crop_poi(self, renditions, crop_name, poi):
+        """
+        Set the crop point of interest in the renditions dictionary for the given crop
+        :param dict renditions
+        :param string crop_name
+        :param dict poi: the point of interest dictionary
+        """
+        fields = ('CropLeft', 'CropTop', 'CropRight', 'CropBottom')
+        if 'x' in poi and 'y' in poi:
+            original_image = renditions['original']
+            crop_data = renditions[crop_name]
+            orig_poi_x = int(original_image['width']*poi['x'])
```

Comment:
if either `original_image['width']` or `poi['x']` is a string this line probably will fail...